CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
-## master
-* Add your own contributions to the next release on the line below this with your name.
+## [1.2](https://github.com/Pinterest/PINOperation/tree/1.2) (2020-06-30)
+
+[Full Changelog](https://github.com/Pinterest/PINOperation/compare/1.1.2...1.2)
+
+**Merged pull requests:**
+
+- Swift Package Manager support [\#32](https://github.com/pinterest/PINOperation/pull/32) ([martinpucik](https://github.com/martinpucik))
```

Comment:
I don't think it's necessary to escape the ‘#`s?

Reply:
This is generated by https://github.com/github-changelog-generator/github-changelog-generator